Cargo.toml

```diff
@@ -48,3 +48,7 @@ rayon = "~1.5"
 version = "1"
 default-features = false
 features = ["user-hooks"]
+
+[dependencies.pyo3]
```

Comment:
Would it be possible to use the `pyo3 = { version = "0.16.0", features = ["auto-initialize"] }` syntax for this dependency and move it up into the `[dependencies]` section. That way it would be easier to see, as a separate section down here one might easily overlook it I feel.